### PR TITLE
Updates due to https://github.com/dask/fastparquet/pull/623

### PR DESCRIPTION
**Pull request by martindurant**
NB: fastparquet has been able to do multi-index for a long time,
don't know why dask raised to this.

- [ ] ~Closes #xxxx~
- [x] Tests added / passed
- [x] Passes `black dask` / `flake8 dask` / `isort dask`

### CODE REVIEW

**Comment by martindurant** [comment]
@dask/maintenance

File: dask/dataframe/io/tests/test_parquet.py [hunk 1]
```diff
@@ -927,6 +923,8 @@ def test_read_parquet_custom_columns(tmpdir, engine):
     ],
 )
 def test_roundtrip(tmpdir, df, write_kwargs, read_kwargs, engine):
+    if engine == "pyarrow":
+        return
```
Comment (rjzamora) [inline dask/dataframe/io/tests/test_parquet.py:927]: Should this be removed?

**Comment by martindurant** [comment]
@rjzamora  , better

File: dask/dataframe/io/tests/test_parquet.py [hunk 2]
```diff
@@ -459,22 +459,18 @@ def test_columns_index_with_multi_index(tmpdir, engine):
...
-        with pytest.raises(ValueError):
-            ddf = dd.read_parquet(fn, engine=engine, index=index.names)
-
     else:
         pq.write_table(pa.Table.from_pandas(df.reset_index(), preserve_index=False), fn)
 
-        # Pyarrow supports multi-index reads
-        ddf = dd.read_parquet(fn, engine=engine, index=index.names)
-        assert_eq(ddf, df)
+    # Pyarrow supports multi-index reads
```
Comment (rjzamora) [inline dask/dataframe/io/tests/test_parquet.py:465]: I guess **both** engines support multi-index reads.

**Review by rjzamora — APPROVED** [review]
LGTM - Thanks for this Martin!

**Pull request merged**